Content.IntegrationTests/Tests/GameRules/NukeOpsTest.cs

```diff
@@ -229,12 +229,15 @@ void CheckDummy(int i)
         var totalSeconds = 30;
         var totalTicks = (int) Math.Ceiling(totalSeconds / server.Timing.TickPeriod.TotalSeconds);
         var increment = 5;
-        var resp = entMan.GetComponent<RespiratorComponent>(player);
+        var hasResp = entMan.TryGetComponent(player, out RespiratorComponent? resp); // Starlight edit
```

Comment:
could just wrap this in a `if (TryComp<RespiratorComponent>(player, out var resp))` instead?